### PR TITLE
Limit text now is hidden when user is searching or filtering

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska 

## Issue
https://github.com/Fliplet/fliplet-studio/issues/6186

## Description
Changed hidden class assignment.

## Screenshots/screencasts
https://streamable.com/39eya5

## Backward compatibility
This change is fully backward compatible.

## Reviewers 
@upplabs-alex-levchenko @YaroslavOvdii

### CODE REVIEW

File: js/layout-javascript/news-feed-code.js [hunk 1]
```diff
@@ -1636,7 +1636,9 @@ DynamicList.prototype.searchData = function(options) {
       }
 
       if (limitEntriesEnabled) {
-        _this.$container.find('.limit-entries-text')[truncated && _this.data.limitEntries > 0 ? 'removeClass' : 'addClass']('hidden');
```
Comment (tonytlwu) [inline js/layout-javascript/news-feed-code.js:1639]: The `truncated` variable from the results of `_this.Utils.Records.runSearch()` should be able to determining this. I believe `truncated` should only be `true` if search, filter and bookmarks are all disabled. There should therefore be no need to separately calculate a `showLimiText` status.

File: js/layout-javascript/news-feed-code.js [hunk 2]
```diff
@@ -1636,7 +1636,9 @@ DynamicList.prototype.searchData = function(options) {
...
+        // Do not show limit text when user is searching or filtering
+        var showLimitText = _this.isSearching || _this.isFiltering || _this.showBookmarks;
+        _this.$container.find('.limit-entries-text').toggleClass('hidden', showLimitText);
```
Comment (tonytlwu) [inline js/layout-javascript/news-feed-code.js:1641]: @upplabs-alex-levchenko @YaroslavOvdii A new linebreak is needed after a block of variable declaration.

```suggestion

        _this.$container.find('.limit-entries-text').toggleClass('hidden', showLimitText);
```

File: js/layout-javascript/news-feed-code.js (hunk 1 above)
Comment (MaksymShokin) [reply js/layout-javascript/news-feed-code.js:1639]: fixed

File: js/layout-javascript/news-feed-code.js (hunk 2 above)
Comment (MaksymShokin) [reply js/layout-javascript/news-feed-code.js]: fixed

**Pull request merged**